go.sum

```diff
@@ -14,6 +14,8 @@ github.com/agext/levenshtein v1.2.1/go.mod h1:JEDfjyjHDjOF/1e4FlBE/PkbqA9OfWu2ki
 github.com/agext/levenshtein v1.2.2 h1:0S/Yg6LYmFJ5stwQeRp6EeOcCbj7xiqQSdNelsXvaqE=
 github.com/agext/levenshtein v1.2.2/go.mod h1:JEDfjyjHDjOF/1e4FlBE/PkbqA9OfWu2ki2W0IB5558=
 github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412/go.mod h1:WPjqKcmVOxf0XSf3YxCJs6N6AOSrOx3obionmG7T0y0=
+github.com/alakae/cloudscale-go-sdk v0.0.0-20200408102057-f2f32c968005 h1:lK1OGJmMgTTtx/A9CzAq1h3ozYp0pQgZVhU5KJSUHIM=
+github.com/alakae/cloudscale-go-sdk v0.0.0-20200408102057-f2f32c968005/go.mod h1:FhOTOCgKAVvRRMQc1mC0D7xK/3zYnmcZBWFXNkacvMc=
```

Comment:
While this is probably temporary, don't forget to make this cloudscale-ch again.